packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -36,35 +38,45 @@ type Props = {
      * provide a container element with 100% width.
      */
     footer: React.Node,
+
+    /** Called when the close button is clicked. */
+    onClickCloseButton?: () => void,
```

Comment:
I think Kevin and I talked about this one and decided to make this name more explicit, though I can't find my reference for this anymore.

I think this prop needs to be optional, because we want folks to be able to declare a modal and pass it to the `ModalLauncher` without specifying their own `onClickCloseButton` behavior, and instead depending on the `ModalLauncher`'s built-in behavior. (See the clone approach we took in #59. In that world, `onClickCloseButton` always is defined by mount time… but it isn't always defined at declaration time.)

In any case, these props definitely need more documentation, no matter what world we choose. Will revisit!

Reply:
Sorry I haven't had a chance to update the spec as per our conversation.  @matchu if have some time today would you mind updating the spec?  If not I should be able to get to it tomorrow.

Reply:
Updated :)